app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/govuk_wrapper.scala.html

```diff
@@ -57,8 +57,8 @@
 }
 
 @headerNavLinks = {
-  @if(withAuth) {
-    <li><a href="@appConfig.signOutUrl" id="signOut">@messages("service.signOut")</a></li>
+  @if(withSignOutLink) {
+    <li><a href="@{appConfig.signOutUrl}?continue=/cgt-property-disposals/start" id="signOut">@messages("service.signOut")</a></li>
```

Comment:
just in case this changes it'd be safer to use `routes.StartController.start().url`

Reply:
done